src/instance.rs

```diff
@@ -480,33 +516,52 @@ impl<'py, T> Borrowed<'_, 'py, T> {
 }
 
 impl<'a, 'py> Borrowed<'a, 'py, PyAny> {
+    /// Constructs a new `Borrowed<'py, PyAny>` from a pointer. Panics if `ptr` is null.
```

Comment:
Good catch. I think I got mixed up because the functions aren't in the same order as for `Bound` - I'll swap them so that `from_ptr` is first at the same time 👍